_docs/tasks/traffic-management-v1alpha3/ingress.md

```diff
@@ -143,14 +143,28 @@ In our case, we have set the selector value to `istio: ingressgateway` to use th
 `istio-ingressgateway` controller. Therefore, to test our `Gateway` we will send requests to
 the `istio-ingressgateway` service.
 
-1. Get the ingressgateway controller pod's hostIP:
+1. Determine the ingressgateway URL:
+
+   * If your cluster is running in an environment that supports external load balancers, use the ingressgateway external address:
+
+   ```command
+   $ kubectl -n istio-system get svc istio-ingressgateway
+   NAME                   TYPE           CLUSTER-IP       EXTERNAL-IP     PORT(S)                                      AGE
+   istio-ingressgateway   LoadBalancer   172.21.232.178   130.211.10.121  80:31380/TCP,443:31390/TCP,31400:31400/TCP   1h
+   ```
+
+   ```command
+   $ export INGRESS_HOST=130.211.10.121
+   ```
+
+   * If load balancers are not supported, use the ingressgateway controller pod's hostIP:
 
    ```command
    $ kubectl -n istio-system get po -l istio=ingressgateway -o jsonpath='{.items[0].status.hostIP}'
    169.47.243.100
    ```
```

Comment:
This instruction doesn't work for IBM Cloud container service, could you add steps for ICS?

Reply:
Strange, I was able to get this working using the LB steps in IKS. Is their some piece that is failing in your cluster?

Reply:
I think the problem is IKS free tier. There a special instructions for it in the bookinfo guide. I guess we need to repeat all those same instructions in here too?
https://github.com/istio/istio.github.io/blob/master/_docs/guides/bookinfo.md#determining-the-ingress-ip-and-port

Reply:
Yes, the host ip needs to be obtained from `bx cs workers {cluster-name}` for node port.  I'm open to suggestion on the best way to refer to it.